internal/vhdx/info.go

```diff
@@ -0,0 +1,233 @@
+// vhdx package adds the utility methods necessary to deal with the vhdx that are used as the scratch
```

Comment:
Any reason to not put these in the vhd package itself?

Reply:
You mean the vhd package inside go-winio?

Reply:
Yes, it seems like we should just keep it together, but I dont have a strong horse in this race :)

Reply:
Had some discussion around this here: https://github.com/microsoft/hcsshim/pull/1850#discussion_r1281154106, I will keep this in internal/vhdx for now, but open to moving it later if required.